src/pages/components/Input/Input.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { forwardRef, InputHTMLAttributes } from 'react';
+
+interface InputPropsInterface extends InputHTMLAttributes<HTMLInputElement> {
+  header: string;
+  type: string;
+}
+
+const Input = forwardRef<HTMLInputElement, InputPropsInterface>(({ header, type = '', ...rest }, ref) => (
+  <div className="flex flex-col w-full my-3 items-left">
+    <h3 className="mb-3 font-normal text-gray-600">{header}</h3>
+    <input
+      // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
good call for adding the comment for now, can you create a follow-up Issue to address removing this comment and stop spreading `rest`?